app/uk/gov/hmrc/pensionschemereturn/connectors/PsrConnector.scala

```diff
@@ -81,8 +81,8 @@ class PsrConnector @Inject()(config: AppConfig, http: HttpClientV2, apiAuditUtil
     psaPspId: String,
     credentialRole: String,
     userName: String
-  )(
-    implicit headerCarrier: HeaderCarrier,
+  )(implicit
```

Comment:
i thought with scala 3 they make you move to given / using syntax?

Reply:
I think we can use both style with scala-3. In future, older version might be deprecated but for now it's fine.

I converted to try new version in this file and it looks working :) 
app/uk/gov/hmrc/pensionschemereturn/models/response/PsrVersionsEtmpResponse.scala